### ISSUE TITLE
Missing aria-label on print close button

### ISSUE THREAD

**Issue by johnnyblasta** [origo-map/origo#1490]
**Describe the bug**
Missing aria-label on print close button.

**Expected behavior**
There should at least be declarative an aria-label on all interactive elements if it's not possible to have other descriptions.
